package.json

```diff
@@ -9,7 +9,7 @@
     "test": "lerna run test",
     "size": "lerna run size",
     "lint": "eslint --ext .ts,.tsx,.js,.jsx packages/",
-    "prettify": "prettier --write '**/*.{js,ts,md}' !**/babel-plugin/test/**",
+    "prettify": "cross-env prettier --write '**/*.{js,ts,md}' !**/babel-plugin/test/**",
```

Comment:
Remove it, please. It must be placed under other PR

Reply:
Done